tests/deploys-images/deploy_test.go

```diff
@@ -17,10 +17,7 @@ var _ = Describe("cOS Deploy tests", func() {
 	AfterEach(func() {
 		// Try to gather mtree logs on failure
 		if CurrentGinkgoTestDescription().Failed {
-			s.GatherLog("/tmp/image-mtree-check.log")
```

Comment:
I'm a bit unsure , maybe I'm missing something, but now we don't catch those anymore?

Reply:
ah, just saw it down below